src/components/RadioButtonGroup/styles.scss

```diff
@@ -0,0 +1,146 @@
+/* stylelint-disable no-descending-specificity,max-line-length */
+@import '../../styles/colors';
+@import '../../styles/shadows';
+@import '../../styles/margins';
+@import '../../styles/paddings';
+@import '../../styles/font-sizes';
+@import '../../styles/border-radius';
+
+%border-margin {
+    border-radius: 0;
+    border-width: 1px;
+    margin-left: -1px;
+}
+
+@mixin user-select() {
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+}
+
+.rainbow-radio-button-group_container {
+    position: relative;
+}
+
+.rainbow-radio-button-group_label {
+    display: inline-block;
+    color: $color-gray-4;
+    font-size: $font-size-text-medium;
+    margin-right: $margin-small;
+    margin-bottom: $margin-xx-small;
+}
+
+.rainbow-radio-button-group_inner-container {
+    // display: inline-block;
+    border-radius: 1.25rem;
+    border: solid 1px #e3e5ed;
+    background-color: rgba(227, 229, 237, 0.4);
+    padding-top: calc(0.1rem + 1px);
+    margin-top: -1px;
+    padding-bottom: 0.1rem;
+    display: -ms-inline-flexbox;
+    display: inline-flex;
+
+    .rainbow-radio-button-group_radio {
+        .rainbow-radio-button-group_radio-label-container {
+            @extend %border-margin;
+
+            border-radius: 1.25rem;
+            padding-top: 0.4rem;
+            padding-right: $padding-large;
+            padding-bottom: calc(0.5rem + 1px);
+            padding-left: $padding-large;
```

Comment:
it is a best practice use the `padding` shortcut when you want defined various paddings